example/transaction.js

```diff
@@ -0,0 +1,22 @@
+import db from './connect'
+
+const insertValues = { first_name: 'Brad', last_name: 'Westfall', datetime_added: 'NOW()' }
+
+Promise.resolve()
+.then(db.beginTransaction)
```

Comment:
If it's no trouble, could you please indent the `.then` lines so it's easier to read?